### DISCUSSION TITLE
Having issues only on Mac Safari browser.

### DISCUSSION THREAD

**Discussion by linups** [livewire/livewire#8899] [1. Help]
Hello guys,

I'm having issues only on MAC Safari browser. Everywhere else it is working. On MAC Safari wire:click is not working...

Here is livewire blade:

`<div>..............
                   <form wire:submit.prevent="submit">
                    @csrf
                    ............
                            <select name="webinarTime" wire:click="getWebinarDates" class="" wire:model="webinarTime">
                                <option>Please select webinar time (required)</option>
                                @if(isset($webinarDates) && !empty($webinarDates))
                                    @foreach($webinarDates as $webinarID => $webinarTime)
                                        <option value="{{$webinarID}}">{{$webinarTime}}</option>
                                    @endforeach
                                @endif
                            </select>
                        </div>
                        ..................
                </form>
............
        </div>
`

And here is component code:
`...................
class OptinModal extends Component
{
    protected $listeners = ['showModal', 'setMyTimezone'];

    public function setMyTimezone($value):void {
        $this->myTimezone = $value;
    }

    public function getWebinarDates():void {
       dd('not working');
    }
..............
}
`

For simplicity i have removed some code... Any ideas what could be wrong? Or how i could debug it? Thank you!

**Comment by fcarboneras** [comment]
hey, did you find a solution why it did not work ?
